packages/core/test/ArrayField_test.js

```diff
@@ -62,6 +62,30 @@ const ExposedArrayKeyTemplate = function(props) {
   );
 };
 
+const CustomOnAddClickTemplate = function(props) {
+  return (
+    <div className="array">
+      {props.items &&
+        props.items.map(element => (
+          <div
+            key={element.key}
+            className="array-item"
+            data-rjsf-itemkey={element.key}>
```

Comment:
Can you remove this attribute? then we should be good

Reply:
Fixed